pages/[id]/index.js

```diff
@@ -14,9 +15,17 @@ const MemberProfile = ({ imageLink, user, contributions, errorMessage }) => {
   const { first_name = '', last_name = '' } = user;
   const memberName = `${first_name} ${last_name} | Member Real Dev Squad`;
 
+  const router = useRouter();
+  const devUser = router.query.dev ? true : false;
```

Comment:
We can make it look more readable using double negation
```javascript
const devUser = !!query.dev
```

Reply:
done